app/brave_main_delegate.cc

```diff
@@ -210,76 +188,13 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
   command_line.AppendSwitchASCII(variations::switches::kVariationsServerURL,
                                  kVariationsServerURL.c_str());
 
-  // Enabled features.
-  std::unordered_set<const char*> enabled_features = {
-    // Upgrade all mixed content
-    blink::features::kMixedContentAutoupgrade.name,
-    password_manager::features::kPasswordImport.name,
-    net::features::kLegacyTLSEnforced.name,
-    // Enable webui dark theme: @media (prefers-color-scheme: dark) is gated
-    // on this feature.
-    features::kWebUIDarkMode.name,
-    blink::features::kPrefetchPrivacyChanges.name,
-    blink::features::kReducedReferrerGranularity.name,
-#if defined(OS_WIN)
-    features::kWinrtGeolocationImplementation.name,
-#endif
-    security_state::features::kSafetyTipUI.name,
-  };
+  // Runtime-enabled features. To override Chromium features default state
+  // please see: brave/chromium_src/base/feature_override.h
+  std::unordered_set<const char*> enabled_features = {};
 
-  // Disabled features.
```

Comment:
JFTR: I've checked all disabled features one by one to confirm all were properly migrated and that seems to be the case :+1: